proposals/incubation/cert-manager.md

```diff
@@ -0,0 +1,138 @@
+
+The [companies-table DevStats dashboard][CompaniesTable] is illustrative. We've seen:
+
+- 4 different companies which have made over a thousand contributions each,
+- 8 different companies which have made over a hundred contributions each,
+- and over a thousand different companies which have contributed something to the project.
+
+In 2021, cert-manager saw [102 new first-time-contributors][NewContributors2021] to the project. Drilling down, cert-manager v1.7 saw at least 7 different individual community contributors, and one of the major features in that release was [contributed by a community member][ReleaseNotes17].
+
+2022 has started strongly too; at the time of writing there have been [21 new contributors][NewContributors2022] already in this calendar year.
+
+In terms of code changes only and ignoring other types of contribution, we see an [average of 10 different individual committers][CommittersCertManager] per month in the 6 months before February 2022 for cert-manager alone, rising to an [average of 20 per month][CommittersAllRepos] across all repositories in the same time period.
+
+### Ongoing Commits
+
+> Demonstrate a substantial ongoing flow of commits and merged contributions
+
+All recent releases of cert-manager have included major features contributed by both the core maintainer team and community members, driven by a contiuous flow of commits every month.
+
+Looking at CNCF DevStats, cert-manager saw an average of [39 pull requests opened][AvgOpenedPRsCertManager] per month in the 6 months before February 2022 (excluding bots), with an average of [30 PRs merged per month][AvgMergedPRsCertManager] in the same time period.
+
+The above numbers only count the main cert-manager repo; as mentioned previously, there are several cert-manager subprojects being regularly worked on too, including the cert-manager website.
+
+Across all repos we see an average of [78 merged PRs][AvgMergedPRsAllRepos] per month in the same time period as above.
+
+### Versioning
+
+> A clear versioning scheme
+
+cert-manager has had a clear versioning scheme and a regular release cadence for over a year. Details can be seen on the cert-manager website under our [Supported Releases][] page, which documents full and clear support lifecycles for the main cert-manager project. We broadly use SemVer.
+
+Being closely tied to Kubernetes, cert-manager also makes great efforts to be clear which versions of Kubernetes itself are supported by a given version of cert-manager.
+
+### Security Process
+
+> Clearly documented security processes explaining how to report security issues
+
+A security reporting mechanism was added in 2021 with a clear [security process][] and a dedicated email address: `cert-manager-security@googlegroups.com`. This security policy applies to all subprojects under the cert-manager umbrella too.
+
```

Comment:
We are missing this part from the security criteria: 

> and describing how the project provides updated releases or patches to resolve security vulnerabilities

We could write a few sentences on our process for backporting and releasing patch versions?

Reply:
I think I'd just kinda assumed they'd read the SECURITY.md file, but I think it would totally be worthwhile to write a little in this doc too. Will add.